src/sagemaker/estimator.py

```diff
@@ -675,8 +677,18 @@ def __init__(
             sagemaker_session=self.sagemaker_session,
         )
 
-        self.profiler_config = profiler_config
-        self.disable_profiler = disable_profiler
+        self.profiler_config = resolve_value_from_config(
+            direct_input=profiler_config,
+            config_path=TRAINING_JOB_PROFILE_CONFIG_PATH,
+            default_value=None,
+            sagemaker_session=self.sagemaker_session,
+        )
+        self.disable_profiler = resolve_value_from_config(
+            direct_input=disable_profiler,
+            config_path=TRAINING_JOB_PROFILE_CONFIG_DISABLE_PROFILER_PATH,
+            default_value=None,
```

Comment:
`default_value` for `disable_profiler` is False.

Reply:
Agreed. And be sure to change the constructor also, from 

```
        disable_profiler: bool = False,
```
into:
```
        disable_profiler: bool = None,
```